cfgov/unprocessed/css/header.less

```diff
@@ -51,10 +51,11 @@
     width: 190px;
     height: 40px;
     margin: 10px;
+    margin-top: 0;
```

Comment:
I'll circle back to this in another PR for the atomic header. Remind me if I forget!